### PR TITLE
Package by pr comment and update @electron/notarize to 2.0.0

### PR DESCRIPTION
**Pull request by yanguoyu**
Refer to https://github.com/Magickbase/neuron-public-issues/issues/199#issuecomment-1607160246

This is the example comment https://github.com/yanguoyu/neuron/pull/49#issuecomment-1612349073.

### CODE REVIEW

**Review by JeffreyMa597 — APPROVED** [review]

File: .github/workflows/package_for_test.yml [hunk 1]
```diff
@@ -1,11 +1,31 @@
 name: Package Neuron for Test
 
 on: 
-  pull_request:
+  issue_comment:
+    types: [created, edited]
   push:
```
Comment (Keith-CY) [inline .github/workflows/package_for_test.yml:6]: I thought that `package for test` would be passive and triggered by a comment on demand. The `push` event here makes each commit packaged, that makes `pacakge by comment` redundant.

Comment (yanguoyu) [reply .github/workflows/package_for_test.yml:6]: The package action trigger by push existed before. So I didn't remove it.

Comment (Keith-CY) [reply .github/workflows/package_for_test.yml:6]: But `package by comment` will be redundant if each commit has its own package

Comment (yanguoyu) [reply .github/workflows/package_for_test.yml:6]: Most of the time, we will not directory push to `Neuron`, we push to our fork repo. `package by comment` is used to package for the pr branch.

Comment (Keith-CY) [reply .github/workflows/package_for_test.yml:6]: > Most of the time, we will not directory push to `Neuron`, we push to our fork repo. `package by comment` is used to package for the pr branch.

Got it

**Review by Keith-CY — APPROVED** [review]

**Review by homura — APPROVED** [review]
LGTM

**Pull request merged**